examples/tictactoe/ts/src/tictactoe-lib.ts

```diff
@@ -0,0 +1,45 @@
+import { Field, PrivateKey, PublicKey, Mina, Party, Signature } from 'snarkyjs';
+import { TicTacToe } from './tictactoe.js';
+
+export function createLocalBlockchain(): PrivateKey[] {
+  let Local = Mina.LocalBlockchain();
+  Mina.setActiveInstance(Local);
+  return [Local.testAccounts[0].privateKey, Local.testAccounts[1].privateKey];
+}
+
+export async function deploy(
+  zkAppInstance: TicTacToe,
+  zkAppPrivkey: PrivateKey,
+  deployer: PrivateKey
+) {
+  const txn = await Mina.transaction(deployer, () => {
```

Comment:
I like that you used deployer as a variable name instead of a generic account variable.